app/models/planning_application.rb

```diff
@@ -279,7 +279,23 @@ def result_present?
   end
 
   def validation_requests
-    (description_change_validation_requests + replacement_document_validation_requests + additional_document_validation_requests + other_change_validation_requests + red_line_boundary_change_validation_requests).sort_by(&:created_at).reverse
+    (replacement_document_validation_requests + additional_document_validation_requests + other_change_validation_requests + red_line_boundary_change_validation_requests).sort_by(&:created_at).reverse
+  end
+
+  def open_description_change_requests
+    description_change_validation_requests.select(&:open?)
```

Comment:
don't think you need this: `description_change_validation_requests.open` does it